package.json

```diff
@@ -41,6 +41,7 @@
     "cookies": "^0.8.0",
     "ethers": "^5.5.0",
     "jsonwebtoken": "^8.5.1",
+    "node-fetch": "^3.2.4",
```

Comment:
import { fetch } from 'undici'; might be a better choice - it's what's powering the experimental built-in fetch in node18

Reply:
I'd make that change if this wasn't a hackathon project, but what's in is working so I'm just gonna merge 😬